tests/ConceptPropertyValueLiteralTest.php

```diff
@@ -221,6 +214,7 @@ public function testGetContainsHtml()
      */
     public function testGetXlLabel()
     {
+        $this->markTestSkipped('disabled since the functionality needs to be reimplemented after the new translation component is in use');
```

Comment:
I think this can be removed now that the translation component is in use.

Reply:
Removed in new commit: https://github.com/NatLibFi/Skosmos/commit/48b31a51acb2fd8be2452c033e0cecd05d86c764